tests/Fixtures/Integration/misc/PHP7_3.test

```diff
@@ -81,6 +82,7 @@ dirname(__FILE__, ); // `dir_constant` rule
 is_null($var, ); // `is_null` rule
 strpos($a, $b, ); // `mb_str_functions` rule
 sample('foo',    'foobarbaz', 'baz'   , ); // `method_argument_space` rule
+$user->setEmail('voff.web@gmail.com', ); // `method_chaining_indentation` rule
```

Comment:
```
$user->setEmail('voff.web@gmail.com', ) // `method_chaining_indentation` rule
               ->setPassword('233434', );
```

now, there is nothing you are chaining here, so rule has nothing to do

Reply:
fixed